tests/test_metrics.py

```diff
@@ -73,3 +66,136 @@ def test_pearson_conf():
     rl, ru = met.pearson_conf(r, n, c=95)
     nptest.assert_almost_equal(rl, np.array([0.406, 0.784]), decimal=3)
     nptest.assert_almost_equal(ru, np.array([0.812, 0.857]), decimal=3)
+
+
+def test_bias():
+    """
+    Test for bias
+    """
+    # example 1
+    x = np.arange(10)
+    y = np.arange(10) + 2
+
+    b_pred = -2
+    b_obs = met.bias(x, y)
+
+    nptest.assert_equal(b_obs, b_pred)
+
+    # example 2
+    x = np.arange(10)
+    y = np.arange(20, 30)
+
+    b_pred = 20
+    b_obs = met.bias(y, x)
+
+    nptest.assert_equal(b_obs, b_pred)
+
+
+def test_aad():
+    """
+    Test for average absolute deviation
+    """
+    # example 1
+    x = np.arange(10)
+    y = np.arange(10) + 2
+    dev_pred = 2
+    dev_obs = met.aad(x, y)
+
+    nptest.assert_equal(dev_obs, dev_pred)
+
+    # example 2, with outlier
+    x = np.arange(10)
+    y = np.arange(10) + 2
+    y[-1] = 201
+    dev_pred = 21
+    dev_obs = met.aad(x, y)
+
+    nptest.assert_equal(dev_obs, dev_pred)
+
+
+def test_mad():
+    """
+    Test for median absolute deviation
+    """
+    # example 1
+    x = np.arange(10)
+    y = np.arange(10) + 2
+    dev_pred = 2
+    dev_obs = met.mad(x, y)
+
+    nptest.assert_equal(dev_obs, dev_pred)
+
+    # example 2, with outlier
+    x = np.arange(10)
+    y = np.arange(10) + 2
+    y[-1] = 201
+    dev_pred = 2
+    dev_obs = met.mad(x, y)
+
+    nptest.assert_equal(dev_obs, dev_pred)
+
+
+def test_rmsd():
+    """
+    Test for rmsd
+    """
+    # example 1
+    x = np.arange(10)
+    y = np.arange(10) + 2
+
+    rmsd_pred = 2
+    rmsd_obs = met.rmsd(x, y)
+
+    nptest.assert_equal(rmsd_obs, rmsd_pred)
+
+    # example 2, with outlier
+    x = np.arange(10)
+    y = np.arange(10) + 2
+    y[-1] = 100
+
+    rmsd_pred = np.sqrt(831)
```

Comment:
This is not correct since the RSS is 8317 which divided by 10 should not be 831. But standard python2 division is of course dangerous and makes this work